tests/features/webtoolsAnalytics.feature

```diff
@@ -0,0 +1,30 @@
+@api @webtoolsAnalytics
+Feature: Check Piwik
+  In order to check if the type attribute is set for the Piwik element.
+  As a developer and or a site administrator
+  I want to check Piwik is available.
+
+  Background:
+    Given these modules are enabled
+      | modules                            |
+      | oe_webtools_analytics              |
+    And I am logged in as a user with the "administrator" role
+
+  Scenario: Check if the PIWIK script is embedded into the page correctly
+    Given I am on the homepage
+    Then the response status code should be 200
+    Then the rendered script should contains the "siteID" with the same value as the configuration set in settings file
```

Comment:
See previous comment:
Then the analytics script should have the SiteID set to 12345

Reply:
I wouldn't override it in the settings file, except during development of the analytics code, or in production. While running the CI we can leave it as-is.